### PR TITLE
refactor: adjust syntax of select from uri.

### PR DESCRIPTION
**Pull request by youngsofun**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

change 

```
`select * from 's3://testbucket/admin/data/parquet/tuple.parquet' (aws_key_id => 'minioadmin', aws_secret_key => 'minioadmin', endpoint_url => 'http://127.0.0.1:9900/', file_format => 'parquet')"`
```

to 

```
select * from 's3://testbucket/admin/data/parquet/tuple.parquet' (connection => (aws_key_id = 'minioadmin', aws_secret_key = 'minioadmin', endpoint_url = 'http://127.0.0.1:9900/')
```

also Closes https://github.com/datafuselabs/databend/issues/13419

- - -
This change is [Reviewable](https://reviewable.io/reviews/datafuselabs/databend/13427)

### CODE REVIEW

File: src/query/ast/tests/it/testdata/statement-error.txt [hunk 1]
```diff
@@ -515,6 +515,18 @@ error:
                 [ copyOptions ]`
 
 
+---------- Input ----------
```
Comment (BohuTANG) [inline src/query/ast/tests/it/testdata/statement-error.txt:518]: Don't understand the error hint and what I will do 😢

Comment (youngsofun) [reply src/query/ast/tests/it/testdata/statement-error.txt:518]: you are right
let me rethink

**Review by BohuTANG — APPROVED** [review]
LGTM

**Comment by BohuTANG** [comment]
Query from stage documents may need modifying with the new as this PR changed, we change the connection option same as copy. cc @soyeric128

**Comment by BohuTANG** [comment]
> Query from stage documents may need modifying with the new as this PR changed, we change the connection option same as copy. cc @soyeric128

I have a tiny update at: https://github.com/datafuselabs/databend/commit/d78c9fd77c794c88ef8a2ef84825385a7e50e157

Please contiue to fix if missing, thanks.

**Pull request merged**